sql/2020/09_Performance/median_lcp_score_by_month.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# Median of Largest contentful paint 75% score by month
+
+SELECT
+  date,
+  device,
+  median_p75_lcp
+FROM (
+  SELECT
+    date,
+    device,
+    PERCENTILE_CONT(p75_lcp, 0.5) OVER(PARTITION BY date, device) AS median_p75_lcp
```

Comment:
I think we can remove the outer SELECT and group by date and device here. Does that still do what we want?

Reply:
Nope..
Column median_p75_lcp contains an analytic function, which is not allowed in GROUP BY at [17:3]